packages/fxa-auth-server/config/dev.json

```diff
@@ -362,7 +363,8 @@
       "enabledClientIds": [
         "98e6508e88680e1a",
         "dcdb5ae7add825d2",
-        "325b4083e32fe8e7"
+        "325b4083e32fe8e7",
+        "5882386c6d801776"
```

Comment:
Luckily I know what this one is, but is there any way we can put comments on these? grumble grumble JSON should have inline comments grumble....

Reply:
at least it's defined in the same file ;)